integrations/segmentio/lib/index.js

```diff
@@ -64,7 +64,9 @@ var Segment = (exports = module.exports = integration('Segment.io')
   .option('saveCrossDomainIdInLocalStorage', true)
   .option('retryQueue', true)
   .option('addBundledMetadata', false)
-  .option('unbundledIntegrations', []));
+  .option('unbundledIntegrations', []))
+  .option('unbundledConfigIds', [])
+  .option('maybeBundledConfigIds', {});
```

Comment:
If we were to conditionally add these arrays in `ajs-renderer` we would still default to empty values here. 🤔

Reply:
Makes sense. I agree.